rimage/transform/parallel_projection.go

```diff
@@ -95,5 +97,162 @@ func (pp *ParallelProjection) PointCloudToRGBD(cloud pointcloud.PointCloud) (*ri
 
 // ImagePointTo3DPoint takes the 2D pixel point and assumes that it represents the X,Y coordinate in mm as well.
 func (pp *ParallelProjection) ImagePointTo3DPoint(pt image.Point, d rimage.Depth) (r3.Vector, error) {
-	return r3.Vector{float64(pt.X), float64(pt.Y), float64(d)}, nil
+	return r3.Vector{X: float64(pt.X), Y: float64(pt.Y), Z: float64(d)}, nil
+}
+
+// ParallelProjectionOntoXZWithRobotMarker allows the creation of a 2D projection of a pointcloud and robot
+// position onto the XZ plane.
+type ParallelProjectionOntoXZWithRobotMarker struct {
+	robotPose *spatialmath.Pose
+}
+
+const (
+	sigmaLevel       = 7
+	imageHeight      = 1080
```

Comment:
Isn't the height & length returned by orbslam currently much smaller than this? Is increasing the image size part of the acceptance criteria?

Reply:
If so, why these values specifically?

Reply:
It is, as some map could be small, for example if the point cloud data is returned in meters, then a room might only take up 10 x 10 pixels. For this version of the jpeg visualization we want to be able to be able to see important features. 

Future work may include the ability to change some to all of these values via the UI. 

These initial constants especially the height and width will probably be changed during the [RSDK-1065](https://viam.atlassian.net/browse/RSDK-1065) ticket to better conform to the UI. 1080 x 1080 was a semi arbitrary choice until then, but is approx the size of the PCD viewer below it

Reply:
That makes sense but if we want the image to be:  1080 X 1080, why is `imageWidth` set to 1070?